cider/tests/functionality/CiderGroupByTest.cpp

```diff
@@ -440,6 +467,128 @@ TEST_F(CiderGroupByPrimitiveTypeMixTest, noConditionGroupByMultiColTest) {
       true);
 }
 
+TEST_F(CiderGroupByVarcharTest, varcharGroupByTest) {
+  /*one not null varchar group by key*/
+  assertQuery(
```

Comment:
can you add some simple cases like this ? `SELECT SUM(col_a), col_c FROM table_test GROUP BY  col_c`

Reply:
added